### PR TITLE
Query and update

### PR DESCRIPTION
**Pull request by m05e5**
# Vet clinic database: query and update animals table
- Add a column species of type string to your animals table. Modify your schema.sql file
- Insert data
- Inside transactions
- insert queries

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
 
 CREATE TABLE animals (
-    id integer AUTO_INCREMENT PRIMARY KEy
+    id integer AUTO_INCREMENT PRIMARY KEY
     name varchar(100)
     date_of_birth date
     escape_attempts integer
     neutered boolean
     weight_kg decimal
+    species varchar(100) DEFAULT
```
Comment (arslanbisharat) [inline schema.sql]: - [ ] [Optional] Let's add a new column of species in this way so the table gets updated completely and we follow the best code practice. 

`ALTER TABLE animals ADD species varchar(255);`

File: data.sql [hunk 2]
```diff
@@ -4,3 +4,9 @@ INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Gabumon', TO_DATE('15/11/2018', 'DD/MM/YYYY'), 8.0, true, 2);
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Pikachu', TO_DATE('07/01/2021', 'DD/MM/YYYY'), 15.04, false, 1);
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Devimon', TO_DATE('12/05/2017', 'DD/MM/YYYY'), 11.0, true, 5);
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Charmander', TO_DATE('08/02/2020', 'DD/MM/YYYY'), -11.0, false, 0);
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Plantmon', TO_DATE('15/09/2022', 'DD/MM/YYYY'), -5.7, true, 2);
```
Comment (arslanbisharat) [inline data.sql]: - [ ] Let's follow the guidelines and update the `Plantmon` date of birth.
Capture

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: data.sql (hunk 2 above)
Comment (m05e5) [reply data.sql]: thanks for your feedback

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉

## Hi @m05e5, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**